src/apps/dashboard/controllers/networking.js

```diff
@@ -142,6 +146,13 @@ export default function (view) {
         page.querySelector('#chkEnableIP6').checked = config.EnableIPv6;
         page.querySelector('#chkEnableIP4').checked = config.EnableIPv4;
         page.querySelector('#txtPublishedServer').value = (config.PublishedServerUriBySubnet || []).join(', ');
+        if (config.PublicUserListingLocalOnly && config.PublicUserListing) {
+            page.querySelector('#selectPublicUserListing').value = 'local';
+        } else if (config.PublicUserListing) {
+            page.querySelector('#selectPublicUserListing').value = 'always';
+        } else {
+            page.querySelector('#selectPublicUserListing').value = 'never';
+        }
```

Comment:
For maintainability: fewer places with the identifier.
```suggestion

        const selectPublicUserListing = page.querySelector('#selectPublicUserListing');

        if (config.PublicUserListingLocalOnly && config.PublicUserListing) {
            selectPublicUserListing.value = 'local';
        } else if (config.PublicUserListing) {
            selectPublicUserListing.value = 'always';
        } else {
            selectPublicUserListing.value = 'never';
        }
```

Reply:
that should be done, thanks for the feedback!